### ISSUE TITLE
Tutorial Documentation needed

### ISSUE THREAD

**Issue by jrober84** [phac-nml/epivizor#3]
Example walk through of the software is needed with a toy dataset

**Comment by kbessonov1984** [comment]
Yes, we can have a github page in `docs` branch where we will have documentation and website deployed on github.io domain. We also have toy dataset included with the server located at https://github.com/phac-nml/epivizor/tree/main/example. The `ecoli_sample_data.csv` is included with the web application and could be downloaded by clicking `Download a demo E.coli dataset` info message
